homeassistant/components/qbittorrent/sensor.py

```diff
@@ -53,32 +55,27 @@
         vol.Required(CONF_URL): cv.url,
         vol.Required(CONF_USERNAME): cv.string,
         vol.Required(CONF_PASSWORD): cv.string,
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_NAME, default=DOMAIN): cv.string,
```

Comment:
Does this change the behavior? Previously it was `qBittorrent`, now it becomes `qbitorrent`,  is this change in behavior made for a reason?